### PR TITLE
[Newton+] added deployment of Sahara dashboard for horizon

### PR DESCRIPTION
**Pull request by sidthe**

### CODE REVIEW

File: chef/cookbooks/horizon/recipes/server.rb [hunk 1]
```diff
@@ -97,6 +97,25 @@
...
+# install horizon sahara plugin if needed
+sahara_ui_pkgname =
+  case node[:platform_family]
+  when "suse"
+    "openstack-horizon-plugin-sahara-ui"
+  when "rhel"
+    "openstack-sahara-ui"
+  end
+
+unless sahara_ui_pkgname.nil?
```
Comment (dirkmueller) [inline chef/cookbooks/horizon/recipes/server.rb]:
```ruby
 if sahara_ui_pkgname
```

Comment (sidthe) [inline chef/cookbooks/horizon/recipes/server.rb]: its consistent with the previously implemented dashboards for manila and magnum.

Comment (sidthe) [inline chef/cookbooks/horizon/recipes/server.rb]: Maybe, however, whats wrong with unless nil?

**Comment by sidthe** [comment]
@dirkmueller done

**Pull request merged**